main/tests/cypress/cypress/integration/create-project/preview_project.spec.js

```diff
@@ -219,4 +219,46 @@ describe(__filename, function () {
     cy.get('table.data-table tr').eq(1).should('to.contain', '15.87');
     cy.get('table.data-table tr').eq(1).should('to.contain', '717');
   });
+
+  /*
+  The test case below uses the ignore feature to test the disable automatic preview update checkbox
+  We first test with automatic preview updates enabled
+  Then, we test with automatic preview updates disabled, which requires the update button to change the preview
+  */
+  it('Tests disabling of automatic preview', function () {
+    navigateToProjectPreview();
+    // **Testing ignore feature with auto preview enabled** //
+    cy.get('input[bind="ignoreInput"]').type('{backspace}1');
+    cy.get('input[bind="ignoreCheckbox"]').check();
+    cy.waitForImportUpdate();
+    // Look for automatic preview update
+    cy.get('table.data-table tr').eq(1).should('to.contain', '1.');
+    cy.get('table.data-table tr').eq(1).should('to.contain', '01002');
+    cy.get('table.data-table tr').eq(1).should('to.contain', 'BUTTER,WHIPPED,WITH SALT');
+    cy.get('table.data-table tr').eq(1).should('to.contain', '15.87');
+    cy.get('table.data-table tr').eq(1).should('to.contain', '717');
+
+    cy.get('input[bind="ignoreCheckbox"]').uncheck();
+    cy.waitForImportUpdate();
+
+    // **Testing ignore feature with auto preview disabled** //
+    cy.get('input[bind="disableAutoPreviewCheckbox"]').check();
+    // Verify no auto update
+    cy.get('input[bind="ignoreCheckbox"]').check();
+    cy.wait(5000); // 5 second wait to be safe
```

Comment:
I would normally avoid using `wait` but I understand that here it is not so easy to do, since we are not waiting for anything to happen in the DOM. We are just waiting to make sure nothing changed even after a while. I am happy with it as it stands but I just wonder if @fgiroud has something else to propose?

Reply:
@johnvl2 Hello and sorry for the late reply
One thing I have noticed is that the button "Ignore first" has no effect alone, you must also enter a number of rows to ignore.
But yes, to have a real test, you have no other choice than to "wait", because the dom is not rendered, and that's maybe something you can add as a comment next to it
Also, you can probably keep only one single assertion, about one single cell of the grid to improve the test readbility

Thx,

Reply:
@fgiroud Yes, I noticed this as well about "ignore first", but with the disable checkbox we can avoid the unnecessary preview update for this as well. Added a comment to the wait call with your explanation above. Also removed unnecessary assertions. Kept just the first two which checks an unchanged cell, and a cell we expect to change to verify proper functionality.